CMakeLists.txt

```diff
@@ -159,6 +159,10 @@ endif()
 if(CORENRN_ENABLE_MPI)
   find_package(MPI REQUIRED)
   add_definitions("-DNRNMPI=1")
+  # avoid linking to C++ bindings
+  add_definitions("-DMPI_NO_CPPBIND=1")
```

Comment:
Just for my information, who uses these? ```grep -r MPI_NO_CPPBIND *``` turns up nothing in my nrn repository that has coreneuron as a submodule.

Reply:
Actually it's in MPI library header. For example, HPE -MPI on BB5 has:

```
# /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/deploy/tools/2020-02-01/linux-rhel7-x86_64/gcc-8.3.0/hpe-mpi-2.21-7pbszh6v5u/include/mpi.h
 * mpio.h must be included after mpi.h
 */
#include "mpio.h"
#include "mpi_t.h"

#if defined(__cplusplus)
}
#if !defined(MPI_NO_CPPBIND)
#include "mpi++.h"
#endif
#endif

```

and that we want to avoid.